### DISCUSSION TITLE
6714f917fa2f0dd3a91d2911

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#3] [General]
# 6714f917fa2f0dd3a91d2911

Discover a wide range of cracked games, mods, and resources at Toxic Games. Download for free and dive into endless gaming adventures!

https://toxicgames.in/6714f917fa2f0dd3a91d2911

**Comment by Admin-ToxicGames** [comment]
![Resident Evil Village](https://img.playbook.com/cwv_LEfmgY-lHg1up9__Lv3ob5zSmHIg-ytz4ArJuew/Z3M6Ly9wbGF5Ym9v/ay1hc3NldHMtcHVi/bGljLzgyOTRmYmY1/LTI1NTUtNGVjNi1i/NDM5LTc4NTk0OWVm/NDJlOA)

### Tested on Macbook Air M1 on High settings, gives around 50-60 FPS🔥.

**Comment by nikaSanikidzeuser** [comment]
tested on air m2, runs well 60-70 fps

**Reply by yashwakulkar** [reply]
really dude , is downloading process is good

**Comment by Chadwick32real** [comment]
hi guys how yall doing

**Reply by yashwakulkar** [reply]
good i am new here

**Comment by Chadwick32real** [comment]
Sorry it’s NakedSnake28 please reach out

**Comment by AnujSwai** [comment]
Send hi @anujswai in telegram

**Comment by 0YEM0** [comment]
my Xbox controller didn't work with the game why?

**Comment by AnujSwai** [comment]
Try changing the setting to input/output to keyboard to controller..enjoy coz by default it set to keyboard and mouse..enjoy

**Reply by 0YEM0** [reply]
where can I change it ?

**Comment by AnujSwai** [comment]
Bro within Game setting for example..go to the desired game find setting and then input setting.you will find keyboard/mouse as input.or this this https://www.youtube.com/watch?v=bylk5VRlJPE

**Reply by 0YEM0** [reply]
THX for your fast feedback but there is no buttons that changes the game input from inside the game. 
Moreover, enjoyable can't be used after the new update on mac. Is there any other solutions?

**Comment by AnujSwai** [comment]
I didnt remember but der is this app paid app which mount joystick keys to keyboard..go for the paid one

**Comment by THAKURabhinavv** [comment]
The application cannot be opened error

**Reply by Admin-ToxicGames** [reply]
Read the instructions and paste the code in terminal.

**Comment by sarthakpujari1260** [comment]
its sayin forbidden when downlaoding with fdm 
tried in incognito mode too
also with removing the extension and simply copy and pasting the link . 
please help

**Reply by Admin-ToxicGames** [reply]
Use Akirabox link, it's working.

**Reply by sarthakpujari1260** [reply]
in akirabox only its saying that and cant downlaod using fdm manager

**Comment by Vergilprotag** [comment]
Bro yeh sb game ps5 controller ko support karenge? Mac pr, mai ek dual sense controller  lene ka soch rha

**Reply by Admin-ToxicGames** [reply]
Yea, not all games but most of em will run.

**Comment by sophia553** [comment]
Hi there!

I’m Sophia and pleased to connect with you. I have a superb opportunity to share with you. As an advertiser, I can offer you some fantastic deals along With Download ads, Pop-ups and Banners with super security systems you’ll love. We are giving maximum prices, and I’m sure you’ll really like it. Let’s explore how we can work together!

BONUS DEAL AWAITING FOR YOUR RESPONSE!

Best Regards,
Sophia
Account Manager

Email: sophia33905241@gmail.com

Telegram: @Sophianaila

**Reply by Admin-ToxicGames** [reply]
No ads currently.

**Comment by sophia553** [comment]
offering you ads, if you need with maximum rates.

**Reply by yashwakulkar** [reply]
hha

**Comment by mooken-mac** [comment]
I tried to open it but they said always app cannot be opened "I used terminal with xattr -cr "

**Reply by MonkeyDLuffy-MOD** [reply]
#### xattr -cr /Applications/Resident\ Evil\ Village.app && codesign --force --deep --sign - /Applications/Resident\ Evil\ Village.app

**Reply by mooken-mac** [reply]
yes I wrote that but it didn't work I don't know why ? I have played many games using latter-cr but with village it didn't work

**Reply by MonkeyDLuffy-MOD** [reply]
that command takes upto 5 mins wait for it to finish, also "xattr -cr" not gonna work on native games, try the above one only.

**Reply by mooken-mac** [reply]
thank you very much

**Comment by AnujSwai** [comment]
Bhai Red dead krdo public, ab paise bhi pure ho gaye acoount bhi bana dia,lekin ab 20 votes chaiye..itna muskil kyu bana rahe ho process

**Reply by yashwakulkar** [reply]
ha bro  read dead 2 ko krdo

**Reply by Admin-ToxicGames** [reply]
No votes needed, I just have to apply a fix on that so I have to re-upload and it takes time due to its size.

**Reply by yashwakulkar** [reply]
Thanks then

**Comment by sophia553** [comment]
Ads offer with good rates available.

**Comment by AnujSwai** [comment]
Once it’s uploaded let us know and one more thing it again showing 950/1200 donation.. please we waited for 6 monthsSent from my iPhoneOn 7 Jul 2025, at 2:28 PM, sophia553 ***@***.***> wrote:﻿
Ads offer with good rates available.

**Comment by amitthakurdogra24-sketch** [comment]
hi i am not able o run the game in mackbook pro 16 2021 m1 pro residential evil village

**Reply by Admin-ToxicGames** [reply]
Error?

**Comment by amitthakurdogra24-sketch** [comment]
its saying application cannot be run

**Reply by Admin-ToxicGames** [reply]
Read Instructions after clicking on the download button.

**Comment by Nicolas1912i** [comment]
Are the DLCs included?

**Reply by Admin-ToxicGames** [reply]
No DLC

**Comment by bhavyavashisth** [comment]
It is sowing me unsupported environment 😭

**Reply by Admin-ToxicGames** [reply]
Enter the correct command in terminal?

**Reply by bhavyavashisth** [reply]
I am using it on Mac a2141
32 gb ram

**Comment by AnujSwai** [comment]
Show me which command you are usingSent from my iPhoneOn 5 Sep 2025, at 8:22 AM, dhanushvramin-art ***@***.***> wrote:﻿
plzz how to solve it

**Comment by AnujSwai** [comment]
I guess this command amd games optimised to M1 base and later models. Please do make Sure they are compatible with Apple Architecture where previous models were intel based.. so compatibility issue is the main problem here

**Comment by Sharafooo** [comment]
Can I play the DLC ?

**Reply by Admin-ToxicGames** [reply]
No

**Reply by Vaibhav-rb** [reply]
will u update in dlc or at least free resident evil 2 ? please
